Automattic/woocommerce-payments
# Extend Travis build matrix with WP/WC version compatibility checks

Extends Travis build and adds WC/WP versions compatibility checks running in master.

#### Changes proposed in this Pull Request

* Adds PHP 7.4 build environment and bumps used WC version 4.3 -> 4.5 as it the most used ATM
* Tests reasonable amunt of WP/WC versions combinations

#### Testing instructions

* Check Travis build for this PR itself: it should include 'compatibility' stage (we are targeting master here)
* After merging into master ensure the merge build has the 'compatibility' stage

-------------------
